model/metrics/system-metrics.yaml

```diff
@@ -494,18 +494,18 @@ groups:
         examples: ["running"]
 
 
-  - id: metric.system.processes.count
+  - id: metric.system.process.count
     type: metric
-    metric_name: system.processes.count
+    metric_name: system.process.count
     brief: "Total number of processes in each state"
     instrument: updowncounter
     unit: "{process}"
     attributes:
-      - ref: system.processes.status
+      - ref: system.process.status
 
-  - id: metric.system.processes.created
+  - id: metric.system.process.created_total
     type: metric
-    metric_name: system.processes.created
+    metric_name: system.process.created_total
```

Comment:
Hum I think this goes against this: [do not use _total](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#do-not-use-total)

Reply:
Interesting point! I wonder though what could be a better naming here since `system.process.created` would indicate the creation time of a process maybe?

Also by `brief: "Total number of processes created over uptime of the host"` and `instrument: counter` I understand that we have to do with monotonically increasing counter but having a delta metric instead of a cumulative value would be confusing so I wonder what is the rational of [do not use _total](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#do-not-use-total) and if it's applicable here. (Sorry if I miss anything here)

Reply:
@open-telemetry/semconv-system-approvers any thoughts on this?

Reply:
`_total` is related to cumulative counters, so if I export to a back-end which works with delta only (by setting up my SDK to export deltas) then it is confusing, that's the reason in the guidelines:

> Counters SHOULD NOT append _total either because then their meaning will be confusing in delta backends.

If I got it right, this metric is measuring the number of process created (also spawn/initiated) since the host has started? Maybe one of those names can be used instead?

Reply:
Thank you @joaopgrassi! I would suggest the following options then:

a) `system.process.created.count`
b) `system.process.spawn`
c) `system.process.num_created`

@open-telemetry/semconv-system-approvers could you provide your feedback/preferences on which one we should go with?

Reply:
I like `system.process.started`. I also like `system.process.created`.

> since `system.process.created` would indicate the creation time of a process maybe?

I don't personally see this as a problem, since it's a metric name (as opposed to an attribute name).

In case it helps, we have similar scenario in JVM metrics with:

* [`jvm.class.count`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/runtime/jvm-metrics.md#metric-jvmclasscount) - current number of classes
* [`jvm.class.loaded`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/runtime/jvm-metrics.md#metric-jvmclassloaded) - number of classes loaded since start
* [`jvm.class.unloaded`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/runtime/jvm-metrics.md#metric-jvmclassunloaded) - number of classes unloaded since start

Reply:
> > since `system.process.created` would indicate the creation time of a process maybe?
> 
> I don't personally see this as a problem, since it's a metric name (as opposed to an attribute name).

Yeah I also agree, I think `system.process.created` as a metric to me would indicate the number of created process over time. I think dropping the `.count` on it makes more sense.

Reply:
Thank's folk! I don't have really strong preference here so I agree we can go with removing the `.count` suffix. I have pushed this change with [141823e](https://github.com/open-telemetry/semantic-conventions/pull/484/commits/141823e00dd5f9e6a1dc869f8d47c3ce8a595e35).

Reply:
I think we are good now! Resolving this